security/src/main/java/org/kaazing/gateway/security/auth/FileLoginModule.java

```diff
@@ -310,11 +301,14 @@ private void getUsernamePassword(boolean useSharedState) throws LoginException {
 
         } catch (IOException e) {
 //            throw (LoginException) (new LoginException(e.getMessage()).initCause(e));
```

Comment:
please remove pointless comments and code when you see them and they are where you are working.